ovos_utterance_corrections_transformer/__init__.py

```
@@ -13,17 +13,31 @@ def __init__(self, name="ovos-utterance-corrections", priority=1):
+                utterances[0], self.db, strategy=self.match_strategy
+            )
+            if conf >= self.confidence_threshold:
                 return [replacement], context
 
-        # replace individual words
+        # Step 2: Apply regex replacements
+        if utterances and self.regex_db:
+            for idx in range(len(utterances)):
+                for pattern, replacement in self.regex_db.items():
+                    try:
+                        utterances[idx] = re.sub(pattern, replacement, utterances[idx])
```

Ruff (0.8.2)

Undefined name `re`

(F821)